ui/src/pages/Configuration/ConfigurationPage.tsx

```diff
@@ -127,12 +138,12 @@ function ConfigurationPage() {
                     </Row>
                 </ColumnLayout>
                 <TabBar activeTabId={activeTabId} onChange={handleChange}>
-                    {tabs.map((tab) => (
+                    {tabs.map((tab: TabSchema) => (
                         <TabBar.Tab key={tab.name} label={_(tab.title)} tabId={tab.name} />
                     ))}
                 </TabBar>
             </div>
-            {tabs.map((tab) => getTabContent(tab))}
+            {tabs.map((tab: TabSchema) => getTabContent(tab))}
```

Comment:
tabs is already typed, let's not pollute the code with unnecessary types, it does not bring the value
```suggestion
            {tabs.map((tab) => getTabContent(tab))}
```

Reply:
done